var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -160,6 +162,8 @@ class Dealii(CMakePackage, CudaPackage):
 
     # Optional dependencies: Packages
     depends_on('adol-c@2.6.4:',    when='@9.0:+adol-c')
+    depends_on('arborx',           when='@9.3:+arborx')
+    depends_on('arborx+trilinos',  when='@9.3:+arborx+trilinos')
```

Comment:
Do we also need to do something like this (to get the kokkos dependency included)
```
conflicts('kokkos', when='@9.3:+arborx+trilinos ^trilinos+kokkos') # Use Trlinos kokkos package
depends_on('kokkos', when='@9.3:+arborx+trilinos ^trilinos~kokkos') # Stand alone package (not sure if the syntax is right)
```
? I'm thinking of the case where kokkos gets pulled in from some other package, maybe when setting up an environment.

Reply:
I think the second check is not necessary, as arborx requires kokkos in trilinos when it pulls in trilinos, i.e., it pulls in trilinos just for kokkos, so if you `spack install arborx+trilinos` you are automatically saying `spack install arborx+trilinos trilinos^kokkos`.

I'll add the first check.

Reply:
Actually, on a second thought, shouldn't also the first check belong to the Trilinos package and not here?

Reply:
> I think the second check is not necessary

Ah yes, you're right.

> shouldn't also the first check belong to the Trilinos package

Yes, I think that it probably does belong there. Should we propose that as a follow up patch?